lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -750,7 +750,9 @@ static FuncDecl *deriveEncodable_encode(DerivedConformance &derived) {
     encodeDecl->getAttrs().add(attr);
   }
 
-  encodeDecl->computeType(FunctionType::ExtInfo().withThrows());
```

Comment:
Ah, these aren't correct.  The `throws` bit goes on the inner function type.

Reply:
You need to store the throws bit _somewhere_ now.

Reply:
The throwing bit gets applied to the ext info based on whether the throwing bit is set in the function decl.  Both the changed places in the Codable conformance synthesis stuff are already setting that.

Reply:
Ah, ok.